project/BensonYang1999/readme.md

```diff
@@ -0,0 +1,57 @@
+# Simple Machine Learning Toolkit
+
+## Basic Information
+A simple library for machine learning, such as regression, classification, clustering.
+
+## Problem to Solve
+I'm trying to build a library in C++ to make machine learning faster and multipurpose.
+
+There are lots of tools for machine learning in python, but they may not be as efficient as using c++. So I'm going to build a machine laarning library in c++ to make it run faster and easily used for C++ users.
+
+
+## Prospective Users
+Anyone who want to use machine learning in either C++ or Python to solve various problems.
+
+
+## System Architecture
+Use `C++` to implement various algorithm
+
+Users can choice using `Python` or `C++` to read data
```

Comment:
Providing API in two languages is a lot of work.  I will suggest you to pick Python because you will be using pybind11 to wrap C++ anyway.

Reply:
OK, I have fixed this problem by only providing python API.